src/main/webapp/partials/instructorHelpSearch.jsp

```diff
@@ -0,0 +1,365 @@
+                                                <div class="profile-pic-icon-click align-center">
+                                                    <a class="student-profile-pic-view-link btn-link">View Photo</a>
+                                                    <img src="" alt="No Image Given" class="hidden">
+                                                </div>
+                                            </td>
+                                            <td>Section A</td>
+                                            <td>Team A</td>
+                                            <td><span class="highlight">Alice</span> Betsy</td>
+                                            <td class="align-center">Joined</td>
+                                            <td><span class="highlight">alice</span>@email.com</td>
+                                            <td class="no-print align-center">
+                                                <a class="btn btn-default btn-xs" title="View details of the student"
+                                                    href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">View</a>
+                                                <a class="btn btn-default btn-xs" title="Use this to edit the details of this student. <br>To edit multiple students in one go, you can use the enroll page: <br>Simply enroll students using the updated data and existing data will be updated accordingly"
+                                                   href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">Edit</a>
+                                                <a class="course-student-delete-link btn btn-default btn-xs"
+                                                title="Delete the student and the corresponding submissions from the course" href="" data-toggle="tooltip" data-placement="top">Delete</a>
+                                                <a class="btn btn-default btn-xs" href="" title="View all data about this student" target="_blank" rel="noopener noreferrer"
+                                                   data-toggle="tooltip" data-placement="top">All Records</a>
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <br>You can search for a student record based on -
+                <ul>
+                    <li>Section name</li>
+                    <li>Team name</li>
+                    <li>Student name</li>
+                    <li>Email</li>
+                </ul>
+                <br>You can search for multiple students based on the attributes mentioned above. For doing that, mention the attribute names in the search box by separating them with spaces.
```

Comment:
Second sentence: `To do so, include the values you wish to search for in the search box, separated by spaces.`